documentation/src/main/wiki/ProductionUsage.md

```diff
@@ -74,11 +75,12 @@ The *core* module contains 3 Knot.x verticle without any sample data. Here's how
           "port": 8080
         }
       ]
-    }
+    },
   }
 }
 ```
 Configuration JSON contains three config sections, one for each Knot.x verticle.
+Each verticle can be configured with additional [Deployment Options](https://github.com/Cognifide/knotx/wiki/GettingStarted#deployment-options) 
```

Comment:
We can get the same link with `[[Deployment Options|GettingStarted#deployment-options]]` .